spec/ics-027-interchain-accounts/packets.proto

```diff
@@ -0,0 +1,19 @@
+syntax = 'proto3';
+
+enum Type {
```

Comment:
You can use `oneof` in proto, it is probably cleaner

Reply:
I think that the all fields in this definition can be used mutually, so I think the enumerator is enough.

Reply:
Hmm, alright